ui/js/component/transactionList/internal/TransactionListItem.jsx

```diff
@@ -6,12 +6,59 @@ import Link from "component/link";
 import lbryuri from "lbryuri";
 
 class TransactionListItem extends React.PureComponent {
+  abandonClaim() {
+    const {
+      claim_id: claimId,
+      claim_name: name,
+      txid,
+      type,
+      nout,
+    } = this.props.transaction;
+    let msg;
+
+    if (type == "tip") {
+      msg = "This will reduce the committed credits to the URL";
+    } else {
+      msg = "This will reclaim you lbc, back to your account";
```

Comment:
Modal body:

Are you sure want to revoke this claim?

This will prevent others from resolving and accessing the content you published. It will return the LBC to your spendable balance, less a small transaction fee.

Reply:
Done.